assignment.js

```diff
@@ -113,6 +129,31 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+function playGame(playerAChoice, playerBChoice){
```

Comment:
@clo-bot  nice approach covering all of the possible outcomes and nice use of using a function here (before we covered them in class)

The approach that you're using involves nested `if` statements and while it works, you will want to avoid nesting if possible. The reason is that nesting can add complexity to your code which makes things more challenging to debug. In this case, the code was still fairly easy to follow.